templates/default/sql/dml/3.3.0.sql.erb

```diff
@@ -6,4 +6,7 @@ REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("jupyt
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("hopsfsmount_apparmor_profile", "<%= node['hops']['docker']['hopsfsmount-apparmor-profile'] %>");
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("apply_hopsfsmount_apparmor_profile_kube", "<%= @apparmor_enabled %>");
 
-REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("ha_enabled", "<%= @ha_enabled %>", 1);
\ No newline at end of file
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("ha_enabled", "<%= @ha_enabled %>", 1);
+
+-- To ensure that enable_kafka_storage_connectors is updated
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("enable_kafka_storage_connectors", "<%= node['hopsworks']['enable_kafka_storage_connectors'] %>", 1);
```

Comment:
You actually don't have to do this. All the DML files are executed at every update. So you just have to change the attributes/defaults.rb file and the value is going to be updated automatically.